gen/definitions/properties.yaml

```diff
@@ -2745,3 +2745,109 @@ vnsLDevIf:
     datasource_required:
       logical_device: "uni/tn-test_tenant_imported_device/lDevVip-test_imported_device"
   custom_test_dependency_name: "ImportedVnsLDevVipWithFvTenant"
+
+infraAccPortP:
+  default_values:
+    parent_dn: "uni/infra"
+
+infraSpAccPortP:
+  default_values:
+    parent_dn: "uni/infra"
+
+infraFexP:
+  default_values:
+    parent_dn: "uni/infra"
+
+infraHPortS:
+  overwrites:
+    type: port_selector_type
+  documentation:
+    type: "The port selector type of the %s object."
+  test_values:
+    resource_required:
+      port_selector_type: "range"
+    datasource_required:
+      port_selector_type: "range"
+    datasource_non_existing:
+      port_selector_type: "range"
+    legacy:
+      port_selector_type: "range"
+    ignore_in_legacy:
+      - "port_selector_type"
```

Comment:
you set the value but then ignore it?

Reply:
Yes the `port_selector_type` is provided, which does a look up for the legacy equivalent of it and then sets this in testvars correctly. The ignore_in_legacy is for the new port_selector_type, which is being set through the required part of tests. Since it has multiple named attributes and the naming is being changed ( which we had not done before ). Decided to change this naming to be consistent acroos the 2 selector resources.